nifi-assembly/pom.xml

```diff
@@ -51,6 +71,30 @@ language governing permissions and limitations under the License. -->
                             </formats>
                         </configuration>
                     </execution>
+                    <execution>
+                        <id>make-extension-manifest-assembly</id>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                        <phase>package</phase>
+                        <configuration>
+                            <finalName>nifi-${project.version}</finalName>
+                            <attach>true</attach>
+                            <archiverConfig>
+                                <defaultDirectoryMode>0775</defaultDirectoryMode>
+                                <directoryMode>0775</directoryMode>
+                                <fileMode>0664</fileMode>
+                            </archiverConfig>
+                            <descriptors>
+                                <descriptor>src/main/assembly/extension-manifests.xml</descriptor>
+                            </descriptors>
+                            <tarLongFileMode>posix</tarLongFileMode>
+                            <formats>
+                                <format>dir</format>
+                                <format>zip</format>
+                            </formats>
```

Comment:
Should the formats be defined in the assembly descriptor file instead?

Reply:
I don't have a strong opinion here, but the reason I put them here and not in the descriptor file was because the other existing execution above this (`make shared resource`) already defines them in the pom, so it seemed consistent. That one was done through the pom config because there are profiles that alter the formats like "dir-only" to save time, but for this case we wouldn't need that so it could be fine to put it in the descriptor.